backend/app/transcribe1.py

```
@@ -674,30 +688,31 @@ async def process_audio_with_groq(audio_path: str, status_queue: asyncio.Queue,
         base_url = f"https://www.youtube.com/watch?v={video_id}"
 
         # Send status update
-        await status_queue.put(json.dumps({"type": "status", "content": "Preparing audio for Groq transcription...", "timestamp": datetime.now().isoformat()}))
+        await status_queue.put(json.dumps({"type": "status", "content": "Preparing audio for cloud transcription...", "timestamp": datetime.now().isoformat()}))
```

Ruff (0.16.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)